src/Console/Arguments.php

```diff
@@ -158,7 +167,12 @@ public function getOption(string $name): string|bool|null
     public function getBooleanOption(string $name): ?bool
     {
         $value = $this->options[$name] ?? null;
-        assert($value === null || is_bool($value));
+        if ($value !== null && !is_bool($value)) {
+            throw new ConsoleException(sprintf(
+                'Value for option `%s` is not of type `bool`, use `getOption()` instead.',
```

Comment:
```suggestion
                'Option `%s` is not of type `bool`, use `getOption()` instead.',
```

I think we can just say the option is not this type.